jenkins/osa_onmetal.functions.groovy

```diff
@@ -347,12 +347,15 @@ def start_api_uptime_tests() {
 
     String host_ip = get_onmetal_ip()
 
-    // run the API uptime tests
+    // run the API uptime tests.  Note that the output directory goes up 2
+    // levels.  That is done to ensure that the output file goes into the root
+    // for this project, rather than buried in the api_uptime/api_uptime
+    // subdirectory.
```

Comment:
But why don't you use an absolute path instead? it would be much safer:
`/root/output`